### PR TITLE
[OBSDEF-6501] tweaking vsphere resources to support multi-cluster env…

### PR DESCRIPTION
**Pull request by ben-schumacher**
…ironments

## Purpose
[OBSDEF-6501](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-6501)

Made changes to VMware platform resources to support environments that have more than one WCP cluster.

## PR checklist
- [x] make test passed
- [x] make build passed
- [ ] helm install <chart> passed
- [x] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
```
Charts:      26 passed, 26 total
Test Suites: 54 passed, 54 total
Tests:       258 passed, 258 total
Snapshot:    0 passed, 0 total
Time:        17.060403473s

```

### CODE REVIEW

**Review by vgupta-mickey — APPROVED** [review]
LGTM

File: objectscale-portal/templates/objectscale-portal-deployment.yaml [hunk 1]
```diff
@@ -46,11 +46,7 @@ spec:
         {{- if (eq .Values.global.platform "VMware") }}
         - name: certificate
           secret:
-            {{- if (eq .Release.Namespace "dellemc-objectscale-system") }}
-            secretName: objectscale-plugin-secret
-            {{- else }}
-            secretName: objectscale-plugin-secret-{{ .Release.Namespace }}
-            {{- end }}
+            secretName: plugin-secret-{{ `{{ .service.prefix }}` }}
```
Comment (Bjohnson131) [inline objectscale-portal/templates/objectscale-portal-deployment.yaml:49]: where is ".service.prefix" defined?

**Review by Bjohnson131 — APPROVED** [review]
I think we may be missing a default value somewhere, but is otherwise good.

**Comment by eddiepavkovic** [comment]
closing.  if we need to pull it in rebase and repull the request

**Pull request closed without merging**